python/servo/package_commands.py

```diff
@@ -133,6 +137,23 @@ def package(self, release=False, dev=False, android=None, debug=False, debugger=
                 checked.update(checking)
                 need_checked.difference_update(checked)
 
+            print("Adding version to Credits.rtf")
+            git_sha = subprocess.check_output([
+                'git', 'rev-parse', '--short', 'HEAD'
+            ]).strip()
+            git_is_dirty = bool(subprocess.check_output([
+                'git', 'status', '--porcelain'
+            ]).strip())
+            version = 'Git Hash: ' + git_sha
+            if git_is_dirty:
+                version += '-dirty'
```

Comment:
Split this logic out of `CommandBase.build_env` into a function so it can be reused (or reuse it some other way).